### ISSUE TITLE
Remove presentation role from pagination

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#4428]
## Problem/Motivation
Dubbot scan revealed that pagination with `<li>` elements had `role="presentation"` added to them, which is incorrect labelling. 

### Describe the bug
`role="presentation"` is added to `<li>` items in pagination

### To Reproduce

1. Go to https://slhs.arizona.edu/about/news
2. Inspect pagination at the bottom of the page
3. Observe `role="presentation"` on the `<li>` item

## Proposed resolution
Remove `role="presentation"`

### Expected behavior
Pagination should still work as expected and pass Dubbot scans.

Send to Jeff Bishop for review.

##  Screenshots
![Image](https://github.com/user-attachments/assets/bc20e525-ff48-4e5f-a8be-99364d2540fe)

## Additional context

- [Dubbot report online](https://app.dubbot.com/a/5d10d4a8482faf70bdd3322d/sites/6850377e5055a30342c1fa8c/pages/68503c8d15852f2da3ad8af7)
- [WCAG 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)
- [DubBot_AccessibilityIssuesDetail_export_2025-06-23.csv](https://github.com/user-attachments/files/20928685/DubBot_AccessibilityIssuesDetail_export_2025-06-23.csv)
- [DubBot_AccessibilityIssuesSummary_export_2025-06-23.csv](https://github.com/user-attachments/files/20928686/DubBot_AccessibilityIssuesSummary_export_2025-06-23.csv)

**Comment by bberndt-uaz** [comment]
I think the best resolution would be using a patch from this Drupal core issue:

 - [Pager template fails accessibility tests for Ellipsis ("...")](https://www.drupal.org/project/drupal/issues/3348552)

**Comment by bberndt-uaz** [comment]
@danahertzberg I'm only seeing the presentation role added to ellipsis pager items. Are you seeing the role on other pager items?
